recipes/geant4/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+declare -a CMAKE_PLATFORM_FLAGS
+if [[ ${HOST} =~ .*darwin.* ]]; then
+	CMAKE_PLATFORM_FLAGS+=(-DCMAKE_OSX_SYSROOT="${CONDA_BUILD_SYSROOT}")
+else
+	CMAKE_PLATFORM_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE="${RECIPE_DIR}/cross-linux.cmake")
+fi
+
+if [[ ${DEBUG_C} == yes ]]; then
+  CMAKE_BUILD_TYPE=Debug
+else
+  CMAKE_BUILD_TYPE=Release
+fi
+
+mkdir geant4-build
+cd geant4-build
+
+cmake                                                          \
+      -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}                   \
+      -DCMAKE_INSTALL_PREFIX=${PREFIX}                         \
+      -DBUILD_SHARED_LIBS=ON                                   \
+      -DGEANT4_INSTALL_EXAMPLES=ON                             \
+      -DGEANT4_INSTALL_DATA=ON                                 \
```

Comment:
Would it make more sense to have the data as a separate package?

Reply:
I don't think so.